src/libraries/System.Data.OleDb/src/ArchitectureSpecificHelpers.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Data.OleDb
+{
+    internal interface ItagDBPROPINFO
```

Comment:
Can we move the contents of this file into `OleDbStruct.cs` itself. That way all the related types will be at the same place.